extensions/mssql/src/objectManagement/ui/objectManagementDialogBase.ts

```diff
@@ -107,6 +107,9 @@ export abstract class ObjectManagementDialogBase<ObjectInfoType extends ObjectMa
 				}
 			}
 		});
+		this.modelView.onValidityChanged((isValid: boolean) => {
```

Comment:
This seems like it should go into ScriptableDialogBase instead, there's where the buttons themselves are actually created.